docs/en/observability/ci-cd-observability.asciidoc

```diff
@@ -0,0 +1,391 @@
+=== CI/CD platform troubleshooting
+
+CI/CD administrators need to assess the impact of anomalies when troubleshooting platform problems quickly,
+whether troubleshooting just one pipeline to much broader outages impacting many pipelines or the entire CI/CD platform.
+
+Elastic Observability enables troubleshooting CI platform outages by providing visualizations of pipeline
+executions as distributed traces, along with the capability to slice and dice pipeline steps in any dimension
+to assess the nature and the impact of the outage.
+
+In the following image, a Jenkins CI build failed, and its exceptions are reported as errors.
+Select any of those errors to view the specific information. In this case, it's errors relating to
+the CI agent that stopped.
+
+[role="screenshot"]
+.Jenkins pipeline build error in Elastic Observability
+image::images/jenkins-pipeline-build.png[Jenkins pipeline builds]
+
+The Errors overview screen provides a high-level view of the exceptions that CI builds catch.
+Similar errors are grouped to quickly see which ones are affecting your services
+and allow you to take action to rectify them.
+
+[role="screenshot"]
+.Jenkins jobs and pipelines errors in Elastic Observability
+image::images/jenkins-pipeline-errors.png[Jenkins pipeline build errors]
+
+[role="screenshot"]
+.Concourse CI pipeline execution as a trace in Elastic Observability
+image::images/concourse-ci-traces.png[Concourse CI traces view]
+
+[discrete]
+[[ci-cd-developers]]
+== Observability for developers
+
+Development teams need to continuously optimize their ever-changing CI/CD pipelines to improve
+their reliability while chasing for faster pipelines. There are requirements to document CI/CD pipelines
+with trace visualizations to show what’s happening and improve performance and reliability
+(flaky tests and pipelines).
+
+Integrating with many popular CIDevOps tools like Maven or Ansible using OpenTelemetry, Elastic Observability
+solves these problems by providing deep insights into the execution of CI/CD pipelines.
+
+[discrete]
+[[ci-cd-visibility]]
+=== CI/CD pipeline visibility
+
+The visualization of CI/CD pipelines as distributed traces in Elastic Observability provides
+documentation and health indicators of all your pipelines.
+
+The APM Service view in Elastic Observability provides a view of all your instrumented CI/CD
+servers.
+
+[role="screenshot"]
+.Jenkins servers in Elastic Observability
+image::images/jenkins-servers.png[Jenkins servers view]
+
+The Service page provides more granular insights into your CI/CD workflows by breaking down health
+and performance metrics by pipeline. You can sort and filter the list to quickly surface which
+pipelines are the slowest or experience the most errors. In the example below, we have sorted pipelines
+by average build duration to show the slowest ones.
+
+[role="screenshot"]
+.A Jenkins server in Elastic Observability
+image::images/jenkins-server.png[Jenkins server view]
```

Comment:
This image is a bit blurry

Reply:
Hi @v1v I used the image from the google doc. Are the fields cropped on that image too? Thanks.

Reply:
I just updated it with a more recent version

Reply:
Thanks!